internal/policies/client.go

```diff
@@ -193,10 +169,13 @@ func (f *Client) getAdmissionPolicies(ctx context.Context, namespace string) ([]
 	return policies.Items, nil
 }
 
-// groupPoliciesByGVRAndLabelSelectorg groups policies by GVR and LabelSelector.
+// groupPoliciesByGVRAndLabelSelectorg groups policies by GVR.
 // If namespaced is true, it will skip cluster-wide resources, otherwise it will skip namespaced resources.
-func (f *Client) groupPoliciesByGVRAndLabelSelector(ctx context.Context, policies []policiesv1.Policy, namespaced bool) (map[schema.GroupVersionResource]map[string][]*Policy, error) {
-	resources := make(map[schema.GroupVersionResource]map[string][]*Policy)
+func (f *Client) groupPoliciesByGVR(ctx context.Context, policies []policiesv1.Policy, namespaced bool) (*Policies, error) { //nolint:funlen
+	policiesByGVR := make(map[schema.GroupVersionResource][]*Policy)
+	auditablePolicies := map[string]struct{}{}
+	skippedPolicies := map[string]struct{}{}
```

Comment:
Can we replace the usage of maps with a [set](https://github.com/deckarep/golang-set)? Just to make things more explicit

Reply:
we could but we would need to introduce a new dependency